aea/cli/utils/click_utils.py

```diff
@@ -196,3 +196,37 @@ def handle_parse_result(
             )
 
         return super().handle_parse_result(ctx, opts, args)
+
+
+def password_option(confirmation_prompt: bool = False, **kwargs) -> Callable:  # type: ignore
+    """Decorator to ask for input if -p flag was provided or use --password to set password value in command line."""
```

Comment:
I think this is OK. Not important, but the only comment I have is that, ideally, `-p` could only be an abbreviation for `--password`, and not behave in a different way.

Some ideas to avoid the above:
1. `--password/-p` is an option that can have zero or one argument; if provided, then use that password, otherwise ask to the user. Basically, like `argparse` with `nargs='?'`. I'm not sure how to implement it with Click though.
2. Have `--password/-p` in combination with another flag, `--interactive/-i` (default: false). If only `--password/-p` is provided, then an argument must be provided (the option). Otherwise (`-i` provided), ask to user.

Reply:
1. in short. can not.
i tried initially. but no success.
it can be flag or option, not both at the same time.
2. the idea was to probide self contain reusable solution, so -p and --password are togheter and i dont to want to introduce also interactive option cause it can be used for something else and be out of scope of the password